client/app/account/components/AccountInfo.tsx

```diff
@@ -2,15 +2,19 @@ import React from 'react'
 import { MdAccountCircle } from 'react-icons/md'
 import { Loading } from '@/_components/layout'
 // import { RouterLink } from '@/_components/ui'
+import { RouterLink } from '@/_components/ui'
 import { colorMap } from '@/_consts'
 import { useGetMeQuery } from '@/_hooks/api'
+import { HouseType } from '@/_types'
 import { DeleteAccount } from '.'
 
 const AccountInfo = () => {
   const { data: user, isLoading } = useGetMeQuery()
 
   if (isLoading || !user) return <Loading />
 
+  const userHouse: HouseType | undefined = user.houses?.[0]
```

Comment:
You don't need `undefined` because you already return on the 14th if it's undefined!

Reply:
This is for a case where a user exists but doesn't yet have a house.

Reply:
Oh okay, in that case, undefined does not make sense so the below is better.

```

  const userHouse = user.houses

      {!userHouse.length && (
        <RouterLink href='/house/create' className='text-teal-600'>
          Create House
        </RouterLink>
      )}
```

In the `DeleteAccount` component, you can check whether the array is empty or not instead of undefined!

Reply:
I got it! Thanks!
2daa9364744489095cfa7a294906c1f4f44d6f48